dduper

```diff
@@ -84,7 +84,7 @@ def get_hashes(out1):
             v.append(idx)
```

Comment:
This fails with new changes. Please use `hashlib.sha256(str(ele).encode('utf-8')).hexdigest()`